autotest/alg/homography.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  Various tests of GDAL homography.
+# Author:   Nathan Olson <nathanmolson at gmail dot com>
+#
+###############################################################################
+# Copyright (c) 2025, Nathan Olson <nathanmolson at gmail dot com>
+#
+# SPDX-License-Identifier: MIT
+###############################################################################
+
+import gdaltest
+import pytest
+
+from osgeo import gdal
+
+
+###############################################################################
+@pytest.fixture(autouse=True, scope="module")
+def module_disable_exceptions():
+    with gdaltest.disable_exceptions():
+        yield
+
+
+###############################################################################
+# Test gdal.InvHomography()
+
+
+def test_homography_1():
+
+    h = (10, 0.1, 0, 20, 0, -1.0, 1.0, 0.0, 0.0)
+    res = gdal.InvHomography(h)
+    expected_inv_h = (-100.0, 10.0, 0.0, 20.0, 0.0, -1.0, 1.0, 0.0, 0.0)
+    for i in range(9):
+        assert res[i] == pytest.approx(expected_inv_h[i], abs=1e-6), res
```

Comment:
pytest assert works on lists (perhaps on tuples too, not sure. So maybe you need to convert to lists first)
```suggestion
    assert res == pytest.approx(expected_inv_h, abs=1e-6), res
```

applies to below instances

Reply:
fixed